<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLSETDialog</class>
 <widget class="QDialog" name="SQLSETDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库设置</string>
  </property>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>271</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>端口号：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="sqlset_sqlfilename"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="sqlset_port"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="sqlset_SQLmodel">
        <item>
         <property name="text">
          <string>MySQL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MariaDB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SQLite</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="sqlset_passwd"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="sqlset_username"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>数据库类型：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>主机名：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="sqlset_hostname"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>数据库文件名：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="sqlset_tryconnect">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>尝试连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sqlset_savesetting">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="sqlset_creattable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>创建表</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="sqlset_inquire">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="sqlset_deletetable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>删除表</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="sqlset_simulationswitch">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>虚拟数据:关</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="sqlset_deleteconfig">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>删除数据库配置</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="sqlset_cleartable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>只清空表数据</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>综合输入窗</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="sqlset_inputerhelp">
            <property name="text">
             <string>帮助</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sqlset_inputclear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="sqlset_inputer">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="sqlset_log"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>651</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>数据库查看器</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="sqlset_sqlread">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sqlset_refreash">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="sqlset_datasqllist">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
